installation.md

```diff
@@ -30,9 +30,17 @@ dep self-update --upgrade
 To install Deployer with Composer, run next command:
 
 ```sh
-composer require deployer/deployer
+composer require deployer/deployer:~4.0 --dev
```

Comment:
Why would you have a deployment package deployed into your production servers? Deployment is a development requirement, not a production one.